ImageServer.Core/Controllers/ImageController.cs

```diff
@@ -28,7 +32,11 @@ public ImageController(IFileAccessService fileService, IImageService imageServic
         [HttpGet("/i/{slug}/{quality:range(0,100)}/{w:range(0,5000)}x{h:range(0,5000)}/{*id}")]
         public async Task<IActionResult> ImageAsync(string id, string slug, int w, int h, int quality, string options = "")
         {
-            Response.Headers.Add("Cache-Control", $"public, max-age={TimeSpan.FromDays(1).TotalSeconds}");
+            if (!int.TryParse(_configuration.GetValue<string>("CacheControlMaxAgeDay"), NumberStyles.Integer, CultureInfo.InvariantCulture, out int maxAgeDay))
+            {
+                maxAgeDay = 1;
```

Comment:
nice, we were handling cache time on varnish /cdn level... was thinking about adding the same but was too lazy.

Reply:
I think if application can handle it, let it go :)